pkg/controllers/settings/settings.go

```diff
@@ -0,0 +1,117 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package settings
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/avast/retry-go"
+	"github.com/samber/lo"
+	"go.uber.org/multierr"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"knative.dev/pkg/logging"
+	"knative.dev/pkg/system"
+
+	"github.com/aws/karpenter-core/pkg/utils/atomic"
+)
+
+const (
+	controllerName = "global-settings"
```

Comment:
thoughts? Obviously nonzero work to change the userfacing name
```suggestion
	controllerName = "operator-settings"
```
```suggestion
	controllerName = "settings"
```